sno/upgrade/__init__.py

```diff
@@ -241,7 +241,103 @@ def upgrade_to_tidy(source):
     tidy_repo.lock_git_index()
     tidy_repo.config["core.bare"] = False
     tidy_repo.config["sno.workingcopy.bare"] = False
-    tidy_repo.write_config(wc_path, is_bare)
+    tidy_repo.write_config(wc_loc, is_bare)
     tidy_repo.activate()
 
     click.secho("In-place upgrade complete: repo is now tidy", fg="green", bold=True)
+
+
+@click.command()
+@click.pass_context
+@click.argument("source", type=click.Path(exists=True, file_okay=False), required=True)
+def upgrade_to_kart(ctx, source):
+    """
+    Upgrade in-place a Sno repository to be a Kart repository.
```

Comment:
Maybe need to clarify this _doesn't_ change any of the stored data (blobs/trees/commits/tags)?

Reply:
Done